src/core/server/core_app/core_app.ts

```diff
@@ -52,6 +52,17 @@ export class CoreApp {
     router.get({ path: '/core', validate: false }, async (context, req, res) =>
       res.ok({ body: { version: '0.0.1' } })
     );
+
+    coreSetup.savedObjects.registerType({
+      name: 'server',
+      hidden: false,
+      namespaceType: 'single',
+      mappings: {
+        properties: {
+          uuid: { type: 'keyword' },
```

Comment:
I wasn't able to find any usages

Reply:
I'm not aware of this being used anywhere either. Probably could remove.

Reply:
Can be done as a separate PR